site/src/pages/WorkspacesPage/WorkspacesEmpty.tsx

```diff
@@ -8,19 +8,76 @@ import { Link } from "react-router-dom";
 export const WorkspacesEmpty = (props: {
   isUsingFilter: boolean;
   templates?: Template[];
+  canCreateTemplate: boolean;
 }) => {
-  const { isUsingFilter, templates } = props;
+  const { isUsingFilter, templates, canCreateTemplate } = props;
   const totalFeaturedTemplates = 6;
   const featuredTemplates = templates?.slice(0, totalFeaturedTemplates);
+  const defaultTitle = "Create a workspace";
+  const defaultMessage =
+    "A workspace is your personal, customizable development environment in the cloud.";
```

Comment:
```suggestion
    "A workspace is your personal, customizable development environment.";
```
I think we can skip "in the cloud."